rosco-web/config/packer/install_packages.sh

```diff
@@ -16,6 +19,49 @@ repository=`echo $repository | sed 's/^"\(.*\)"$/\1/'`
 # Also convert a comma-separated list to a whitespace-separated one.
 packages=`echo $packages | sed 's/^"\(.*\)"$/\1/' | sed 's/,/ /g'`
 
+function install_jq() {
+  if [[ "$package_type" == "deb" ]]; then
+    sudo apt-get update
+    sudo DEBIAN_FRONTEND=noninteractive apt-get install --force-yes -y jq
+  fi
+
+  if [[ "$package_type" == "rpm" ]]; then
+    sudo yum -y install jq
+  fi
+}
+
+function remove_jq() {
```

Comment:
What if jq was already installed before this script ran?

Reply:
Good point---it's probably unlikely it would be on a newly-provisioned image, but I shouldn't assume this.  It should be easy to check this before doing the install/uninstall.